rocketpy/simulation/flight.py

```diff
@@ -3074,17 +3074,13 @@ def _calculate_pressure_signal(self):
                 "Pressure - Without Noise (Pa)",
                 "linear",
             )
-            parachute.noisy_pressure_signal_function = Function(
-                parachute.noisy_pressure_signal,
-                "Time (s)",
-                "Pressure - With Noise (Pa)",
-                "linear",
-            )
             parachute.noise_signal_function = Function(
                 parachute.noise_signal, "Time (s)", "Pressure Noise (Pa)", "linear"
             )
-
-        return None
+            parachute.noisy_pressure_signal_function = (
+                parachute.clean_pressure_signal_function
+                + parachute.noise_signal_function
+            )
```

Comment:
This `Function` should be identical to `Function(parachute.noisy_pressure_signal,"Time (s)","Pressure - With Noise (Pa)","linear",)`.

Can you guarantee that it is?

Reply:
For each parachute, the `noisy_pressure_signal` is defined as the sum of the clean_pressure_signal and the noise_signal itself. That's a definition, should be reasonable.

The modified code is doing the very same operation as before.